sentry-uitest/sentry-uitest-android-benchmark/src/androidTest/java/io/sentry/uitest/android/benchmark/SentryBenchmarkTest.kt

```diff
@@ -0,0 +1,137 @@
+package io.sentry.uitest.android.benchmark
+
+import android.content.Context
+import android.view.Choreographer
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.core.app.launchActivity
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.espresso.action.ViewActions.swipeUp
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.runner.AndroidJUnitRunner
+import io.sentry.ITransaction
+import io.sentry.Sentry
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.uitest.android.benchmark.util.BenchmarkOperation
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.assertTrue
+
+@RunWith(AndroidJUnit4::class)
+class SentryBenchmarkTest {
+
+    private lateinit var runner: AndroidJUnitRunner
+    private lateinit var context: Context
+    private lateinit var choreographer: Choreographer
+
+    @BeforeTest
+    fun setUp() {
```

Comment:
Should likely call `Sentry.close` first since we use the `SentryAndroid.init` in tests?

Reply:
Yep, but directly in the test that inits the sdk (the other doesn't do it)